src/timed-stream.ts

```diff
@@ -65,7 +65,8 @@ export class TimedStream extends PassThrough {
     super({
       ...options,
       objectMode: true,
-      highWaterMark: 0,
+      readableHighWaterMark: 0, // We need to disable readside buffering to allow for acceptable behavior when the end user cancels the stream early.
+      writableHighWaterMark: 0, // We need to disable writeside buffering because in nodejs 14 the call to _transform happens after write buffering. This creates problems for tracking the last seen row key.
```

Comment:
I think this doesn't change the way the stream works, but its nicer if we have the comments here that we had for readRows.